curriculum/challenges/english/25-front-end-development/lab-blog-post-card/66eaddd04a9e533fba689001.md

```diff
@@ -0,0 +1,239 @@
+    - You should an have an `a` with a class of `read-more`. Assign the text `Read More` to it.
+4. The `blog-post-card` class should add a white background, rounded corners, shadow for depth, overflow hidden to round the image borders, and control the card width and text alignment.
+5. The `post-img` class should ensure that the image fills the card's width and adds a border at the bottom.
+6. The `post-content` class should add padding inside the card for spacing.
+7. The `post-title` and `post-excerpt` elements within `post-content` should be styled for text color and margin.
+8. The `read-more` element should be styled for text color, background color, margin, `inline-block` display, `border-radius`, and `padding`. It should be styled like a button.
+9. The `read-more` element should have a hover effect that changes the background color.
+
+# --hints--
+
+You should have a `.blog-post-card` `div`.
+
+```js
+assert.exists(document.querySelector("div.blog-post-card"));
+```
+
+You should have a `.post-img` `img`. Make sure your image has an `alt` text.
+
+```js
+const img = document.querySelector('img.post-img');
+assert.exists(img, 'The <img> element with class "post-img" exists');
+assert.isString(img.alt, 'The alt text of the <img> element is a string');
+assert.isNotEmpty(img.alt, 'The alt text of the <img> element is not empty');
+```
+
+You should have a `.post-content` `div`.
+
+```js
+assert.exists(document.querySelector("div.post-content"));
+```
+
+You should have a `.post-title` `h2`.
+
+```js
+assert.exists(document.querySelector("h2.post-title"));
+```
+
+You should have a `.post-excerpt` `p`.
+
+```js
+assert.exists(document.querySelector("p.post-excerpt"));
+```
+
+You should have a `.read-more` `a`.
+
+```js
+assert.exists(document.querySelector("a.read-more"));
+```
+
+You should have a `.read-more` `a` should have a `Read More` text.
+
+```js
+const link = document.querySelector('a.read-more');
+assert.exists(link, 'The <a> element with class "read-more" exists');
+assert.strictEqual(link.textContent, 'Read More', 'The <a> element has the correct text');
+```
+
+`blog-post-card` should have a `border-radius` property with a value (should not be 0 or a negative value).
+
+```js
+const card = document.querySelector('.blog-post-card');
+assert.exists(card, 'The element with class "blog-post-card" exists');
+
+const borderRadius = getComputedStyle(card).borderRadius;
+assert.notStrictEqual(borderRadius, '0px', 'The card has rounded corners');
+```
+
+`blog-post-card` should have a white background.
+
+```js
+const card = document.querySelector('.blog-post-card');
+assert.exists(card, 'The element with class "blog-post-card" exists');
+
+const backgroundColor = getComputedStyle(card).backgroundColor;
+const whiteColors = ['rgb(255, 255, 255)', 'rgba(255, 255, 255, 1)', '#ffffff', '#fff', 'white'];
+
+assert.include(whiteColors, backgroundColor.toLowerCase(), 'The card has a white background');
```

Comment:
you can use `oneOf` here, as you are testing `backgroundColor`, it's the thing that should be all on the left
```suggestion
assert.oneOf(backgroundColor.toLowerCase(), whiteColors, 'The card has a white background');
```